files/default/lib/bumpzone.rb

```diff
@@ -0,0 +1,115 @@
+#!/opt/chef/embedded/bin/ruby
+require 'git'
+require 'json'
+require 'octokit'
+
+# Library to bump DNS zones
+class BumpZone
+  @commit_msg = "\n\n"
+  @warning_msg = "Warning:\n\n"
+  @warning = false
+  ONELINE_REGEX = /([\w\d\.-_]+\s+[\d]+\s+IN\s+SOA.*[\w\d\.-_]+\s+)(\d{10})(\s+[\d]+\s+[\d]+\s+[\d]+\s+[\d]+)/
+  MULTILINE_REGEX = /([\w\d\.-_]+\s+IN\s+SOA\s+[\w\d\.-_]+\s+[\w\d\.-_]+\s+\(\n\s+)(\d{10})(.*\n.*\n.*\n.*\n.*\n\s+\))/
```

Comment:
And if you'd like a quick example of a SOA record using a hypen: `db.10.in-addr.arpa:10.in-addr.arpa.	2560	IN	SOA	ns1.auth.osuosl.org. hostmaster.10.in-addr.arpa. 1486695035 16384 2048 1048576 2560`